.chloggen/mx-psi_system-detector-host-id.yaml

```diff
@@ -0,0 +1,30 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
I wonder whether this is a breaking change or rather a fix to a previous breaking change 🤔

Reply:
Since this has been out for a few months now I would rather mark this as breaking in case users have adapted to the new behavior